_includes/phone.html

```diff
@@ -1,14 +1,28 @@
 {% assign lang = site.active_lang %}
 
 <div
-  class="crt-phone display-none desktop:display-flex flex-align-center bg-primary-darker text__reverse radius-bottom-md position-absolute pin-right pin-top margin-right-4 padding-x-3 padding-y-2"
+  class="
```

Comment:
Is there a line below class and before all the classes?

Reply:
Maybe format this a little nicer?  Whatever you think is reasonable.

Reply:
I'll go with Prettier.